nimsuggest/nimsuggest.nim

```diff
@@ -659,8 +654,6 @@ proc processCmdLine*(pass: TCmdLinePass, argv: openArray[string]; conf: ConfigRe
         gMode = mepc
         conf.verbosity = compVerbosityMin  # Port number gotta be first.
       of "debug": conf.incl optIdeDebug
-      of "v2": conf.suggestVersion = 0
-      of "v1": conf.suggestVersion = 1
```

Comment:
Please keep the `v2` option for now (by treating it as a no-op). Other external tools, such as Saem's extension, still use it.